### ISSUE TITLE
plot_gg fails with theme_void

### ISSUE THREAD

**Issue by jebyrnes** [tylermorganwall/rayshader#54]
Hello! Been trying to do some work with theme_void to void out (haha) axes and such, and keep getting the same error.  Here's a reprex - and this works for other scales, too.

```
library(rayshader)
library(ggplot2)

a_plot <- ggplot(iris,
       aes(x = Sepal.Length, y = Sepal.Width, 
           color = Species)) +
  geom_point() + 
  theme_void() 

plot_gg(a_plot, multicore=TRUE,
          width=4,
          height=4,
          scale=250)
```

This produces

```
Error in (function (el, elname)  : 
  Element legend.box must be a string or numeric vector.
```

**Comment by tylermorganwall** [comment]
This particular case cannot be mapped to 3D--the `Species` column is a factor, which is not a continuous scale and thus doesn't map to 3D.

There was a bug due to the theme_void, however, so I'll include the fix in the next version. Thanks for the report!

**Comment by tylermorganwall** [comment]
Fixed in 1954bc85c151f536b25058b6152bb09bdc67c54b

**Comment by jebyrnes** [comment]
Ah. Had initially run into this with a continuous scale, and wanted to file a reprex - must have been the earlier error you referred to!
